packages/flutter/test/material/back_button_test.dart

```diff
@@ -180,4 +180,35 @@ void main() {
     ));
     expect(iconText.text.style.color, Colors.red);
   });
+
+  testWidgets('CloseButton onPressed overrides default pop behavior', (WidgetTester tester) async {
+    bool closePressed = false;
```

Comment:
Nit: A slightly more descriptive variable name
```suggestion
    bool customCallbackWasCalled = false;
```

Reply:
Done, thanks.